tools/importers/helpers/modifiers.ts

```diff
@@ -0,0 +1,22 @@
+const camelize = (str: string) => {
+	return str
+		.replace(/(?:^\w|[A-Z]|\b\w)/g, function (word, index) {
+			return index === 0 ? word.toLowerCase() : word.toUpperCase();
+		})
+		.replace(/\s+/g, '');
+};
+
+const makeHexShortcut = (hex: string) => {
```

Comment:
it's color specific. IMO, it deserves a separate file

Reply:
Done.